packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,122 @@
+import fs from "fs";
+import path from "path";
+import inquirer from "inquirer";
+import { run, RunOptions } from "npm-check-updates";
+import { installDependencies } from "../utils/installDependencies.js";
+import { Index } from "npm-check-updates/build/src/types/IndexType.js";
+
+function getPackageJSON(projectPath: string) {
```

Comment:
Please use the utility functions here for reading and writing JSON files:

https://github.com/triggerdotdev/trigger.dev/blob/main/packages/cli/src/utils/fileSystem.ts